### DISCUSSION TITLE
weekly/2023-07-28

### DISCUSSION THREAD

**Discussion by giscus** [ATQQ/sugar-blog#115] [Announcements]
# weekly/2023-07-28

① Arc 浏览器！！！ ② 前端代理神器 whistle 客户端 ③ 显示页面重定向历史chrome插件 ④ clang.js - 支持在浏览器里执行C/C++代码 ⑤ JS minification benchmarks ⑥ Bun v0.7.0 支持Vite ⑦ ChatGPT 国粹版 ⑧ ChatGPT 安卓版APP上线

http://sugarat.top/weekly/2023-07-28.html

**Comment by javaworldcn** [comment]
1
